spark/src/main/scala/org/apache/comet/DataTypeSupport.scala

```diff
@@ -37,7 +37,7 @@ trait DataTypeSupport {
 
   private def isGloballySupported(dt: DataType): Boolean = dt match {
     case ByteType | ShortType
-        if CometSparkSessionExtensions.isComplexTypeReaderEnabled(SQLConf.get) &&
+        if CometSparkSessionExtensions.usingDataFusionParquetExec(SQLConf.get) &&
```

Comment:
I wonder if there ways to disable specific for unit rather than disabling for all bytes and shorts, but should be in a separate PR

Reply:
@parthchandra can answer this one. We did discuss this.

Reply:
Short answer: no I don't think that is possible here.
The DataType here is a Spark type which only has ByteType and ShortType. Both are signed types (Spark has no unsigned types). Spark support reading Parquet unsigned types into Spark signed types so both uint_8/uint_16 and int_8/int_16 get read into ByteType/ShortType.